### PR TITLE
Create Posts with a specified language, or fallback

### LINKED ISSUE #781: 🚀 [Feature] Language of Contribution
## :rocket: Feature

The User should be able to select the Language of the Contribution.

### Design & Layout

Alpha - Create Post - Language - Bildschirmfoto 2019-06-06 um 09 13 14

### PR DESCRIPTION
**Pull request by mattwr18**
### Issues

- fixes #781 

### Todo

- [ ] Write component tests for languages

### CODE REVIEW

**Comment by mattwr18** [comment]
Screenshot at 2019-06-12 19:40:21
Screenshot at 2019-06-12 19:40:21
Screenshot at 2019-06-12 19:42:19

**Comment by mattwr18** [comment]
Screenshot at 2019-06-13 17:27:40

this is what you would expect @Tirokk??

File: webapp/components/ContributionForm/index.vue [hunk 1]
```diff
@@ -137,7 +138,10 @@ export default {
       this.$refs.contributionForm.update('content', value)
     },
     returnLocaleName(locale) {
-      if (this.$i18n.locale() === locale.code) {
+      if (
+        (this.contribution && this.contribution.language === locale.code) ||
+        (!this.contribution && this.$i18n.locale() === locale.code)
+      ) {
```
Comment (Tirokk) [inline webapp/components/ContributionForm/index.vue]: What happens in the else case?

Comment (mattwr18) [reply webapp/components/ContributionForm/index.vue]: good question @Tirokk, I didn't think there was a case where `this.$i8n.locale()` is undefined :thinking: 
what would you expect to happen if there is no locale?

Comment (Tirokk) [reply webapp/components/ContributionForm/index.vue]: Just use english as the world language @mattwr18 , I think. Or?
If the else case is not possible, we don’t need an ìf` here?!

Comment (mattwr18) [reply webapp/components/ContributionForm/index.vue]: Addressed in our latest pairing session @Tirokk... unfortunately, I forgot to add you as a co-author, but I think you know I appreciate you!!

**Pull request merged**